src/SprykerEco/Zed/Adyen/Business/AdyenFacadeInterface.php

```diff
@@ -0,0 +1,131 @@
+        PaymentMethodsTransfer $paymentMethodsTransfer,
+        QuoteTransfer $quoteTransfer
+    ): PaymentMethodsTransfer;
+
+    /**
+     * Specification:
+     * - Handle Authorize OMS command, make request to API, process response.
+     *
+     * @api
+     *
+     * @param \Orm\Zed\Sales\Persistence\SpySalesOrderItem[] $orderItems
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     * @param array $data
+     *
+     * @return void
+     */
+    public function handleAuthorizeCommand(
+        array $orderItems,
+        OrderTransfer $orderTransfer,
+        array $data
+    ): void;
+
+    /**
+     * Specification:
+     * - Handle Cancel OMS command, make request to API, process response.
+     *
+     * @api
+     *
+     * @param \Orm\Zed\Sales\Persistence\SpySalesOrderItem[] $orderItems
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     * @param array $data
+     *
+     * @return void
+     */
+    public function handleCancelCommand(
+        array $orderItems,
+        OrderTransfer $orderTransfer,
+        array $data
+    ): void;
+
+    /**
+     * Specification:
+     * - Handle Capture OMS command, make request to API, process response.
+     *
+     * @api
+     *
+     * @param \Orm\Zed\Sales\Persistence\SpySalesOrderItem[] $orderItems
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     * @param array $data
+     *
+     * @return void
+     */
+    public function handleCaptureCommand(
+        array $orderItems,
+        OrderTransfer $orderTransfer,
+        array $data
+    ): void;
+
+    /**
+     * Specification:
+     * - Handle Refund OMS command, make request to API, process response.
+     *
+     * @api
+     *
+     * @param \Orm\Zed\Sales\Persistence\SpySalesOrderItem[] $orderItems
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     * @param array $data
+     *
+     * @return void
+     */
+    public function handleRefundCommand(
+        array $orderItems,
+        OrderTransfer $orderTransfer,
+        array $data
+    ): void;
+
+    /**
+     * Specification:
+     * - Saves order payment method data according to quote and checkout response transfer data.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Generated\Shared\Transfer\SaveOrderTransfer $saveOrderTransfer
+     *
+     * @return void
+     */
+    public function saveOrderPayment(QuoteTransfer $quoteTransfer, SaveOrderTransfer $saveOrderTransfer): void;
+
+    /**
+     * Specification:
+     * - Executes make payment request to API.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Generated\Shared\Transfer\CheckoutResponseTransfer $checkoutResponse
+     *
+     * @return void
+     */
+    public function executePostSaveHook(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponse): void;
```

Comment:
You should rename the method: the name should say what it does (sends make payment request).

Reply:
It does a lot of things. I'll keep it as is, just update specification to show what it does.